app/code/local/Mobbex/Mobbex/Helper/Sdk.php

```diff
@@ -0,0 +1,27 @@
+<?php
+
+class Mobbex_Mobbex_Helper_Sdk extends Mage_Core_Helper_Abstract
+{
+    /** @var Mobbex_Webpay_Helper_Instantiator */
+    public $instantiator;
+
+    public function __construct() {
+        \Mage::helper('mobbex/instantiator')->setProperties($this, ['settings', 'helper']);
+    }
+
+    /**
+     * Allow to use SDK classes.
+     */
+    public function init()
+    {
+        // Set platform information
+        \Mobbex\Platform::init('magento', $this->helper::VERSION, Mage::getBaseUrl(),
+        [
+            'magento' => Mage::getVersion(),
+            'sdk'     => class_exists('\Composer\InstalledVersions') ? \Composer\InstalledVersions::getVersion('mobbexco/php-plugins-sdk') : '',
```

Comment:
Aún falta agregar la comprobación que agregamos en abril:
https://github.com/mobbexco/woocommerce/blob/01e2ad553c3d5de9dce4409c12438f602c1a3547/mobbex-for-woocommerce.php#L97C71-L97C71